projects/graph/src/graph.py

```diff
@@ -6,4 +6,42 @@
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
-        pass  # TODO
+        self.graph = {}
+    
+    def get_vertices(self):
+        return [int(k) for k in self.graph]
+        
+    def add_vertex(self, vertex):
+        if vertex not in self.graph: 
+            self.graph[vertex] = {}
+    
+    def add_edge(self, vertex, edge):
+        if edge in self.graph:
+            if self.graph[vertex] == {}:
```

Comment:
using `set` will prob make your life easier.  You don't have to worry about duplicates and you can just the `.add(edge)`